src/hotspot/share/cds/cdsHeapVerifier.cpp

```diff
@@ -124,6 +126,13 @@ CDSHeapVerifier::CDSHeapVerifier() : _archived_objs(0), _problems(0)
   ADD_EXCL("sun/invoke/util/ValueConversions",           "ONE_INT",                // E
```

Comment:
At line 46 the example that explains how the verifier works discusses save and restore of field `Foo#archivedFoo`. I believe that field was supposed to be declared as `static` but it is actually declared as an instance field.

Reply:
I edited the comment to add `static` to the declaration of `Foo#archivedFoo`.